# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/tests/browser/browser_hasbeforeunload.js
# Commit: 4b76024db21c
# Full Hash: 4b76024db21cff006a7424a35c412c6e0a153eb0
# Author: Kris Maglione <maglione.k@gmail.com>
# Date: 2020-09-22 03:47:42
# Regressor Bug: 1655866
# File Overlap Count: 1
# Description:
#   Bug 1655866: Part 5 - Use native PermitUnload implementation from front-end code. r=nika,mconley
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D88318
# ==============================================================================

diff -r 55fb069319f1 -r 4b76024db21c dom/tests/browser/browser_hasbeforeunload.js
--- a/dom/tests/browser/browser_hasbeforeunload.js	Mon Sep 21 22:41:01 2020 +0000
+++ b/dom/tests/browser/browser_hasbeforeunload.js	Mon Sep 21 22:41:04 2020 +0000
@@ -395,11 +395,11 @@
  *        True if hasBeforeUnload is expected to be true.
  */
 function assertHasBeforeUnload(browser, expected) {
-  Assert.equal(browser.frameLoader.remoteTab.hasBeforeUnload, expected);
+  Assert.equal(browser.hasBeforeUnload, expected);
 }
 
 /**
- * Tests that the nsIRemoteTab hasBeforeUnload attribute works under
+ * Tests that the MozBrowser hasBeforeUnload property works under
  * a number of different scenarios on inner windows. At a high-level,
  * we test that hasBeforeUnload works properly during page / iframe
  * navigation, or when an <iframe> with a beforeunload listener on its